src/common/tcp_info.h

```diff
@@ -0,0 +1,30 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2024 Clyso GmbH
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
```

Comment:
Is this license compatible with Ceph?  @jdurgin ?

Reply:
Sorry, copy pasted the wrong header 🤦. I'll update the file with the default.
Is there a canonical template for file headers with license?

Reply:
I don't know our license story, but Mark / Dan most likely could answer that

Reply:
@anthonyeleven this is what Ceph is mostly licensed with. Please refer to the `COPYYING` file. You'll be able to read

```
Files: *
Copyright: (c) 2004-2010 by Sage Weil <sage@newdream.net>
License: LGPL-2.1 or LGPL-3 (see COPYING-LGPL2.1 and COPYING-LGPL3)
```

Additionally, this being in the context of ceph common, makes perfect sense to be licensed as LGPL.